--- origsrc/clutter-1.14.6/examples/Makefile.am	2013-11-20 14:24:01.000000000 -0600
+++ src/clutter-1.14.6/examples/Makefile.am	2014-01-28 21:24:54.081300800 -0600
@@ -37,7 +37,7 @@ LDADD = \
 AM_CFLAGS = $(CLUTTER_CFLAGS) $(GDK_PIXBUF_CFLAGS) $(MAINTAINER_CFLAGS)
 
 AM_CPPFLAGS = \
-	-DTESTS_DATADIR=\""$(abs_top_srcdir)/tests/data"\" \
+	-DTESTS_DATADIR=\""`cygpath -m $(abs_top_srcdir)/tests/data`"\" \
 	-DG_DISABLE_SINGLE_INCLUDES \
 	-DGLIB_DISABLE_DEPRECATION_WARNINGS
 
--- origsrc/clutter-1.14.6/tests/conform/Makefile.am	2013-11-20 14:24:01.000000000 -0600
+++ src/clutter-1.14.6/tests/conform/Makefile.am	2014-01-28 21:25:19.106732200 -0600
@@ -163,7 +163,7 @@ test_conformance_CPPFLAGS = \
 	-DCOGL_ENABLE_EXPERIMENTAL_API \
 	-DG_DISABLE_DEPRECATION_WARNINGS \
 	-DCLUTTER_DISABLE_DEPRECATION_WARNINGS \
-	-DTESTS_DATADIR=\""$(top_srcdir)/tests/data"\"
+	-DTESTS_DATADIR=\""`cygpath -m $(top_srcdir)/tests/data`"\"
 
 test_conformance_CFLAGS = -g $(CLUTTER_CFLAGS)
 
--- origsrc/clutter-1.14.6/tests/interactive/Makefile.am	2013-11-20 14:24:01.000000000 -0600
+++ src/clutter-1.14.6/tests/interactive/Makefile.am	2014-01-28 21:25:28.648277900 -0600
@@ -150,7 +150,7 @@ test_interactive_SOURCES = test-main.c $
 nodist_test_interactive_SOURCES = test-unit-names.h
 test_interactive_CFLAGS = $(CLUTTER_CFLAGS) $(GDK_PIXBUF_CFLAGS)
 test_interactive_CPPFLAGS = \
-	-DTESTS_DATADIR=\""$(abs_top_srcdir)/tests/data"\" \
+	-DTESTS_DATADIR=\""`cygpath -m $(abs_top_srcdir)/tests/data`"\" \
 	-DG_DISABLE_SINGLE_INCLUDES \
 	-DGLIB_DISABLE_DEPRECATION_WARNINGS \
 	-DCLUTTER_DISABLE_DEPRECATION_WARNINGS
--- origsrc/clutter-1.14.6/tests/micro-bench/Makefile.am	2013-11-20 14:24:01.000000000 -0600
+++ src/clutter-1.14.6/tests/micro-bench/Makefile.am	2014-01-28 21:25:43.066102600 -0600
@@ -21,7 +21,7 @@ AM_CFLAGS = \
 	-DG_DISABLE_SINGLE_INCLUDES \
 	-DGLIB_DISABLE_DEPRECATION_WARNINGS \
 	-DCLUTTER_DISABLE_DEPRECATION_WARNINGS \
-	-DTESTS_DATA_DIR=\""$(top_srcdir)/tests/data/"\"
+	-DTESTS_DATA_DIR=\""`cygpath -m $(top_srcdir)/tests/data/`"\"
 
 LDADD = $(common_ldadd) $(CLUTTER_LIBS) -lm
 
--- origsrc/clutter-1.14.6/tests/performance/Makefile.am	2013-11-20 14:24:01.000000000 -0600
+++ src/clutter-1.14.6/tests/performance/Makefile.am	2014-01-28 21:25:51.363577200 -0600
@@ -24,7 +24,7 @@ AM_CFLAGS = \
 	-DG_DISABLE_SINGLE_INCLUDES \
 	-DGLIB_DISABLE_DEPRECATION_WARNINGS \
 	-DCLUTTER_DISABLE_DEPRECATION_WARNINGS \
-	-DTESTS_DATA_DIR=\""$(top_srcdir)/tests/data/"\"
+	-DTESTS_DATA_DIR=\""`cygpath -m $(top_srcdir)/tests/data/`"\"
 
 perf-report: check
 
